### PR TITLE
added abi version for electron 6

### PR DESCRIPTION
**Pull request by patrickrobbins**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -69,7 +69,8 @@ var supportedTargets = [
   {runtime: 'electron', target: '3.0.0', abi: '64', lts: false},
   {runtime: 'electron', target: '4.0.0', abi: '64', lts: false},
   {runtime: 'electron', target: '4.0.4', abi: '69', lts: false},
-  {runtime: 'electron', target: '5.0.0', abi: '70', lts: false}
+  {runtime: 'electron', target: '5.0.0', abi: '70', lts: false},
+  {runtime: 'electron', target: '6.0.0', abi: '73', lts: false}
```
Comment (MarshallOfSound) [inline index.js]: `6.0.0` is still in beta, it should be added to `futureTargets` below

Comment (patrickrobbins) [reply index.js]: 10-4

File: index.js [hunk 2]
```diff
@@ -70,6 +70,7 @@ var supportedTargets = [
   {runtime: 'electron', target: '4.0.0', abi: '64', lts: false},
   {runtime: 'electron', target: '4.0.4', abi: '69', lts: false},
   {runtime: 'electron', target: '5.0.0', abi: '70', lts: false}
+
```
Comment (malept) [inline index.js]:
```suggestion
```

**Comment by dsanders11** [comment]
@MarshallOfSound, @malept, can this or #65 be merged? 6.0.0 is on beta 9 now, and it's a pain to use the beta without this change. Not much point in having a `futureTargets` if it doesn't get used for something like the betas.

**Comment by malept** [comment]
Sorry, I don't have commit rights to this repository.

**Comment by issacgerges** [comment]
Can raise a separate issue on `electron-rebuild` but it also may be worthwhile to have an optional `--abi` argument there that could be used to unblock folks in cases like this. Happy to do the work if it's something you want @MarshallOfSound

**Comment by MarshallOfSound** [comment]
@issacgerges Maybe something like `--force-abi=78` or similar, the ABI number is only used to de-dupe / determine when to rebuild 👍

**Comment by dsanders11** [comment]
Thanks @MarshallOfSound and @lgeiger for pushing it through.

**Pull request merged**